MIGRATION.md

```diff
@@ -1,296 +1,714 @@
 
-- `withSentryApi` (Replacement: `wrapApiHandlerWithSentry`)
-- `withSentryAPI` (Replacement: `wrapApiHandlerWithSentry`)
-- `withSentryGetServerSideProps` (Replacement: `wrapGetServerSidePropsWithSentry`)
-- `withSentryGetStaticProps` (Replacement: `wrapGetStaticPropsWithSentry`)
-- `withSentryServerSideGetInitialProps` (Replacement: `wrapGetInitialPropsWithSentry`)
-- `withSentryServerSideAppGetInitialProps` (Replacement: `wrapAppGetInitialPropsWithSentry`)
-- `withSentryServerSideDocumentGetInitialProps` (Replacement: `wrapDocumentGetInitialPropsWithSentry`)
-- `withSentryServerSideErrorGetInitialProps` was renamed to `wrapErrorGetInitialPropsWithSentry`
-- `nextRouterInstrumentation` (Replaced by using `browserTracingIntegration`)
-- `IS_BUILD`
-- `isBuild`
+`@sentry/hub` has been removed and will no longer be published. All of the `@sentry/hub` exports have moved to
+`@sentry/core`.
 
-## Removal of `Span` class export from SDK packages
+#### @sentry/tracing
 
-In v8, we are no longer exporting the `Span` class from SDK packages (e.g. `@sentry/browser` or `@sentry/node`).
-Internally, this class is now called `SentrySpan`, and it is no longer meant to be used by users directly.
+`@sentry/tracing` has been removed and will no longer be published. See
+[below](./MIGRATION.md/#3-removal-of-deprecated-apis) for more details.
 
-## Removal of Severity Enum
+For Browser SDKs you can import `BrowserTracing` from the SDK directly:
 
-In v7 we deprecated the `Severity` enum in favor of using the `SeverityLevel` type. In v8 we removed the `Severity`
-enum. If you were using the `Severity` enum, you should replace it with the `SeverityLevel` type. See
-[below](#severity-severitylevel-and-severitylevels) for code snippet examples
+```js
+// Before
+import * as Sentry from '@sentry/browser';
+import { BrowserTracing } from '@sentry/tracing';
 
-## Removal of the `Offline` integration
+Sentry.init({
+  dsn: '__DSN__',
+  tracesSampleRate: 1.0,
+  integrations: [new BrowserTracing()],
+});
 
-The `Offline` integration has been removed in favor of the offline transport wrapper:
-http://docs.sentry.io/platforms/javascript/configuration/transports/#offline-caching
+// After
+import * as Sentry from '@sentry/browser';
 
-## Removal of `enableAnrDetection` and `Anr` class (##10562)
+Sentry.init({
+  dsn: '__DSN__',
+  tracesSampleRate: 1.0,
+  integrations: [new Sentry.BrowserTracing()],
+});
+```
 
-The `enableAnrDetection` and `Anr` class have been removed. See the
-[docs](https://docs.sentry.io/platforms/node/configuration/application-not-responding/) for more details how to migrate
-to `anrIntegration`, the new integration for ANR detection.
+If you were importing `@sentry/tracing` for the side effect, you can now use `Sentry.addTracingExtensions()` to add the
+tracing extensions to the SDK. `addTracingExtensions` replaces the `addExtensionMethods` method from `@sentry/tracing`.
 
-## Removal of `Sentry.configureScope` (#10565)
+```js
+// Before
```

Comment:
super-l: doesn't have to happen now but for clarity, we could label the changes as `v7/v8` vs. `before/after`?

Reply:
yes good idea!